main.py

```diff
@@ -6,23 +6,32 @@
 # Global variable used for storing data (bad practice)
 data = []
 
+
 # Inefficient data manipulation
 @app.post("/add_item")
 def add_item(item: str):
     global data
     data.append(item)
     return {"message": "Item added successfully"}
```

Comment:
**Suggestion:** Avoid using global variables as it is a bad practice. Instead, consider using a class to encapsulate the data and its related operations.
```suggestion
class DataStore:
    def __init__(self):
        self.data = []

    def add_item(self, item: str):
        self.data.append(item)
        return {"message": "Item added successfully"}

data_store = DataStore()

@app.post("/add_item")
def add_item(item: str):
    return data_store.add_item(item)
```